modules/ROOT/pages/anypoint-platform-cli-commands.adoc

```diff
@@ -1221,7 +1283,7 @@ Besides the default `--help`, `-f`/`--fields` and `-o`/`--output` options, this
 |===
 |Command | Description |  Example
 |`--name` | New asset name .2+<.<| `exchange asset modify --name newName --tags test,sample`
-|`--tags` | Comma-separated tags for the asset
+|`--tags` | Comma-separated tags for the asset |
```

Comment:
```suggestion
|`--tags` | Comma-separated tags for the asset 
```
Remove the extra pipe character (`|`)
Otherwise the table will display incorrectly.